apps/omg_eth/lib/eth/root_chain.ex

```diff
@@ -345,7 +399,23 @@ defmodule OMG.Eth.RootChain do
            {:ok,
             Enum.map(logs, fn log ->
               decode_in_flight_exit_challenged(log)
-              |> Map.put(:call_data, get_in_flight_exit_challenged_data(log["transactionHash"]))
+              |> Map.put(
+                :call_data,
+                get_call_data(
+                  log["transactionHash"],
+                  "challengeInFlightExitNotCanonical",
```

Comment:
we have inconsistency ... once we use `"func_name(args)"` and then somewhere else `"func_name"`, cannot this be refactored so we always use the second notation?